### PR TITLE
feat(bookmarks): Add bookmarks endpoints based on user-item API

### PR DESCRIPTION
**Pull request by Matovidlo**

### CODE REVIEW

File: interaction_bookmark.go [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+	return Request{
+		Path:   "/bookmarks/",
+		Method: http.MethodPost,
+		Params: params,
+	}
+}
+
+// DeleteBookmark deletes a bookmark uniquely specified by userId, itemId, and timestamp or
+// all the bookmarks with the given `userId` and `itemId` if `timestamp` is omitted.
```
Comment (dbmtv) [inline interaction_bookmark.go]:
```suggestion
// DeleteBookmark deletes a bookmark uniquely specified by `userId`, `itemId`, and `timestamp` or
// all the bookmarks with the given `userId` and `itemId` if `timestamp` is omitted.
```

**Review by dbmtv — COMMENTED** [review]
Please mention in the README that bookmarks are supported.

File: interaction_bookmark.go (hunk 1 above)
Comment (Matovidlo) [reply interaction_bookmark.go]: fixed.

**Comment by Matovidlo** [comment]
I have updated also README.md because I forget to update it.

**Review by dbmtv — APPROVED** [review]

**Pull request merged**